### PR TITLE
[Hotfix] Remove profile picture edit functionality

### PR DESCRIPTION
**Pull request by karlolee50**
### Definition of Done
- [x] Profile picture functionality is removed
- [x] Picture display is removed
- [x] User should not be able to change profile picture
- [x] User should not be able to upload profile picture

#### Pages Affected
- http://localhost:3003/profile
- http://localhost:3003/admin/profile

### Scenarios/ Test Cases
- [x] Profile picture functionality is removed
- [x] Picture display is removed
- [x] User should not be able to change profile picture
- [x] User should not be able to upload profile picture

### Screenshots
image

image

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #156 
- [x] Both user admin and student could not change/upload profile picture
- [x] Profile picture functionality is removed
------------------------------------
hotfix-1
hotifx-2

-----------------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/student/profile/ProfileDetail/index.js [hunk 1]
```diff
@@ -282,7 +209,7 @@ const ProfileDetail = () => {
               </div>
             </center>
           )}
```
Comment (Teraaay) [inline src/pages/student/profile/ProfileDetail/index.js:211]: Please remove if not needed, thank you

File: src/pages/student/profile/ProfileDetail/index.js [hunk 2]
```diff
@@ -282,7 +209,7 @@ const ProfileDetail = () => {
...
-        </Form>
+        </Form> */}
```
Comment (Teraaay) [inline src/pages/student/profile/ProfileDetail/index.js]: Please remove the commented code if not needed, thank you

**Comment by Teraaay** [comment]
Review time: 5 minutes
Comment/s: 1

**Pull request merged**